.circleci/config.yml

```diff
@@ -360,6 +360,11 @@ jobs:
       - NODE_VERSION: << pipeline.parameters.node-version >>
     steps:
       - build-win
+      - run:
+          name: 'Bundle the extensions'
+          shell: bash.exe
+          command: |
+            npm run vscode:bundle
```

Comment:
this was required for the int tests to operate properly when commands were called across extensions which we do in a few cases.  I'm don't 100% understand why it's required, but I'm going to find out.